api/src/main/webapp/WEB-INF/queue.yaml

```diff
@@ -46,3 +46,15 @@ queue:
   retry_parameters:
     task_retry_limit: 1
     task_age_limit: 5m
+
+- name: createWorkspaceQueue
+  target: api
+
+  # rate parameters
+  bucket_size: 500
+  rate: 1/s
+  max_concurrent_requests: 10
+
+  retry_parameters:
+    task_retry_limit: 1
```

Comment:
Retrying this could cause complications - probably set to 0?

Looks like 0 == no retry, had to confirm this on SO.. https://stackoverflow.com/a/58282049

Reply:
good catch, thanks